adafruit_lsm6ds/__init__.py

```diff
@@ -64,55 +64,19 @@
 
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_LSM6DSOX.git"
+
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_LSM6DS.git"
+
```

Comment:
Shouldn't the original lines be replaced?

Reply:
yeup!